app/i18n/id/conf.php

```diff
@@ -12,305 +12,305 @@
-			'scroll' => 'while scrolling (except for important feeds)',	// TODO
-			'upon_gone' => 'when it is no longer in the upstream news feed',	// TODO
-			'upon_reception' => 'upon receiving the article',	// TODO
-			'when' => 'Mark an article as read…',	// TODO
-			'when_same_title' => 'if an identical title already exists in the top <i>n</i> newest articles',	// TODO
+			'article_open_on_website' => 'ketika artikel dibuka di situs web aslinya',
+			'article_viewed' => 'ketika artikel dilihat',
+			'focus' => 'saat difokuskan (kecuali untuk feed penting)',
+			'keep_max_n_unread' => 'Jumlah maksimum artikel yang tetap belum dibaca',
+			'scroll' => 'saat menggulir (kecuali untuk feed penting)',
+			'upon_gone' => 'saat tidak lagi ada di feed berita atas',
+			'upon_reception' => 'saat menerima artikel',
+			'when' => 'Tandai artikel sebagai telah dibaca…',
+			'when_same_title' => 'jika judul identik sudah ada di <i>n</i> artikel terbaru',
 		),
 		'show' => array(
-			'_' => 'Articles to display',	// TODO
-			'active_category' => 'Active category',	// TODO
-			'adaptive' => 'Adjust showing',	// TODO
-			'all_articles' => 'Show all articles',	// TODO
-			'all_categories' => 'All categories',	// TODO
-			'no_category' => 'No category',	// TODO
-			'remember_categories' => 'Remember open categories',	// TODO
-			'unread' => 'Show only unread',	// TODO
+			'_' => 'Artikel untuk ditampilkan',
+			'active_category' => 'Kategori aktif',
+			'adaptive' => 'Penyesuaian tampilan',
+			'all_articles' => 'Tampilkan semua artikel',
+			'all_categories' => 'Semua kategori',
+			'no_category' => 'Tidak ada kategori',
+			'remember_categories' => 'Ingat kategori yang terbuka',
+			'unread' => 'Hanya tampilkan yang belum dibaca',
 		),
-		'show_fav_unread_help' => 'Applies also on labels',	// TODO
-		'sides_close_article' => 'Clicking outside of article text area closes the article',	// TODO
+		'show_fav_unread_help' => 'Berlaku juga pada label',
+		'sides_close_article' => 'Klik di luar area teks artikel untuk menutup artikel',
 		'sort' => array(
-			'_' => 'Sort order',	// TODO
-			'newer_first' => 'Newest first',	// TODO
-			'older_first' => 'Oldest first',	// TODO
+			'_' => 'Urutan penyortiran',
+			'newer_first' => 'Terbaru dulu',
+			'older_first' => 'Terlama dulu',
 		),
-		'sticky_post' => 'Stick the article to the top when opened',	// TODO
-		'title' => 'Reading',	// TODO
+		'sticky_post' => 'Tempelkan artikel di bagian atas saat dibuka',
+		'title' => 'Membaca',
 		'view' => array(
-			'default' => 'Default view',	// TODO
-			'global' => 'Global view',	// TODO
-			'normal' => 'Normal view',	// TODO
-			'reader' => 'Reading view',	// TODO
+			'default' => 'Tampilan default',
+			'global' => 'Tampilan global',
+			'normal' => 'Tampilan normal',
+			'reader' => 'Tampilan membaca',
 		),
 	),
 	'sharing' => array(
 		'_' => 'Sharing',	// TODO
 		'add' => 'Add a sharing method',	// TODO
```

Comment:
There are a few more TODOs here, if you can :-)

Reply:
i'll do it next 2 days, i tryin to do that before but i still can't focus xD, need to complete all my assessment first

Reply:
Merging this PR already. Please make a new one for the few remaining things when you find time :-)